programs/test/udp_proxy.c

```diff
@@ -139,18 +139,7 @@ static struct options
     unsigned int seed;          /* seed for "random" events                 */
 } opt;
 
-static void exit_usage( const char *name, const char *value )
```

Comment:
There is no reason to touch `udp_proxy.c`as far as I can see. If the changes are not needed, let's revert them.